index.html

```diff
@@ -57,7 +57,7 @@ <h2>The Main Program</h2>
         </div>
         <div class="program-wrap">
             <div class="program">
-                <img src="./images/lecture_icon.png" alt="">
+                <img src="./images/keynote1.png" alt="">
                 <h3>Keynote</h3>
```

Comment:
- [ ] 👏 Wonderful improvements have been made to this section for the mobile version based on the previous reviewer's feedback. However, there's a small adjustment that's still needed. To ensure optimal readability, I recommend reducing the size of the titles so that they no longer overlap with the paragraphs. Currently, there's an overlap issue on screens with a width of 375px. Addressing this detail will contribute to a seamless and visually appealing user experience. Keep up the great work! 🌟📝🚀

| Your | Design |
| --- | --- |
|  ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/c6642d98-e248-4c3e-a52f-36be5dc54e63) | ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/86b73fa6-5a52-4174-9961-1300e9a92e55)|